brainlit/utils/session.py

```diff
@@ -236,6 +240,7 @@ def pull_vertex_list(
         img = self.pull_bounds_img(bounds)
         return img, bounds, vox_in_img_list
 
+    # TOMMY REVIEW
     def pull_chunk(
```

Comment:
I think you mean, pull a "subvolume" of an image around a point, not "chunks"

Reply:
No, I actually mean chunks